x11-libs/libxft/libxft-2.3.2.recipe

```diff
@@ -0,0 +1,99 @@
+SUMMARY="X.Org Xft library"
+DESCRIPTION="The X.Org project provides an open source implementation of the \
+X Window System. \
+The development work is being done in conjunction with the freedesktop.org \
+community. \
+The X.Org Foundation is the educational non-profit corporation whose \
+Board serves this effort, and whose Members lead this work."
+HOMEPAGE="https://www.x.org/"
+COPYRIGHT="1989-95 GROUPE BULL
+	1998 Arnaud LE HORS
+	1996 Lorens Younes"
+# TODO: check all the licenses in the COPYING file in the next century.
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libXft-$portVersion.tar.gz"
+CHECKSUM_SHA256="26cdddcc70b187833cbe9dc54df1864ba4c03a7175b2ca9276de9f05dce74507"
+SOURCE_DIR="libXft-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86_gcc2 x86"
+
+PROVIDES="
+	libxft$secondaryArchSuffix = $portVersion
+	lib:libxft$secondaryArchSuffix = 4.11.0 compat >= 4
+	"
+
```

Comment:
The newline here should be removed, as required by the [Guidelines][0]

```diff
-
```

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering